src/episodes.jl

```diff
@@ -98,10 +90,3 @@ function Base.append!(e::Episodes, x)
         push!(e.inds, (lengthe.episodes, i))
     end
 end
```

Comment:
How does one pushes, say, a new state to `Episodes` ? Perhaps we need a `push!(e::Episodes, x::NamedTuple)` that will push x to the last episode of the vector of Episode.

Reply:
yes, I didn't consider too much about `Episodes` in this PR yet.

Reply:
Added